testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/federation/ldap/LDAPUserMultipleCredentialTest.java

```diff
@@ -0,0 +1,173 @@
+package org.keycloak.testsuite.federation.ldap;
+
+
+import org.jboss.arquillian.graphene.page.Page;
+import org.junit.*;
```

Comment:
Coud you please remove star imports?

Reply:
Replaced imports with absolute class imports